src/main/webapp/app/overview/course-score-calculation.service.ts

```diff
@@ -51,7 +51,7 @@ export class CourseScoreCalculationService {
                         if (score == undefined) {
                             score = 0;
                         }
-                        pointsAchievedByStudentInCourse += score * this.SCORE_NORMALIZATION_VALUE * maxPointsReachableInExercise;
+                        pointsAchievedByStudentInCourse += round(score) * this.SCORE_NORMALIZATION_VALUE * maxPointsReachableInExercise;
```

Comment:
wouldn't it be better to round the whole statement? 

```suggestion
                        pointsAchievedByStudentInCourse += round(score * this.SCORE_NORMALIZATION_VALUE * maxPointsReachableInExercise);
```